docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -44,17 +44,31 @@ Make sure your Pi is flashed with a Viam-compatible operating system, and that y
 Before programming the Pi to make the plant watering robot functional, you need to physically set up the plant watering robot by wiring the different components together.
 You will set up the robot to receive signals from the capacitive soil moisture sensor and signal to the peristaltic pump when it is time to pump water from the water's container to the plant's container.
 
+### Full Wiring Diagram
+
+![The full wiring diagram for all the hardware for the Plant Watering Robot.](../../img/plant-watering-pi/full-wiring.png)
+
 ### Wire your ADC
 
 The analog-to-digital converter (ADC) between your capacitive soil moisture sensor and Pi converts the analog signals created by the capacitive soil moisture sensor's readings to digital signals that can be processed by your Pi, which expects digital signals to come to it through its GPIO pins.
```

Comment:
```suggestion
The analog-to-digital converter (ADC) converts the capacitive soil moisture sensor's analog readings to digital signals that can be processed by your Pi, which expects digital signals to come to it through its GPIO pins.
```
[out of diff] I noticed this sentence is quite long. Another option is to try and split it into two sentences.